elements/pfe-band/demo/index.html

```diff
@@ -193,9 +199,11 @@ <h2>Darkest band with header tag</h2>
       </header>
       <section class="body1 generic1 band">
       <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-      <pfe-cta pfe-priority="secondary" slot="pfe-band--footer">
+<footer slot="pfe-band--footer">
```

Comment:
I would suggest leaving at least 1 band in this demo page using the original mark-up approach because it surfaces a bug in the band layout/cta component that stretches it inside that slot. We want to eventually fix that and it would be good to have it top-of-mind in the demo.  Also, the original syntax is the preferred approach for assigning a CTA to the footer slot.